package.json

```diff
@@ -30,18 +31,21 @@
     "enzyme": "^3.9.0",
     "enzyme-adapter-react-16": "^1.12.1",
     "jest-css-modules-transform": "^2.1.1",
-    "prop-types": "^15.7.2",
+    "joi-browser": "^13.4.0",
     "jwt-decode": "^2.2.0",
+    "prop-types": "^15.7.2",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
     "react-redux": "^7.0.3",
     "react-router-dom": "^4.3.1",
     "react-test-renderer": "^16.8.6",
+    "react-toastify": "^5.1.0",
     "redux": "^4.0.1",
     "redux-devtools-extension": "^2.13.8",
     "redux-logger": "^3.0.6",
     "redux-thunk": "^2.3.0",
     "semantic-ui-react": "^0.86.0",
+    "toastify": "^1.0.12",
```

Comment:
I don't think we used this package in our implementation for this feature. we can uninstall it if we don't need it anymore..